packs/twitter/actions/update_status.yaml

```diff
@@ -0,0 +1,11 @@
+---
+  name: "update_status"
+  runner_type: "run-python"
+  description: "Update your status (post a new tweet)."
+  enabled: true
+  entry_point: "update_status.py"
+  parameters:
+    status:
+      type: "string"
```

Comment:
Yep, looks like it does:

* https://github.com/Julian/jsonschema/blob/master/jsonschema/validators.py#L52
* https://github.com/Julian/jsonschema/blob/master/jsonschema/compat.py#L30